Add notification

lkorasik opened the pull request.

lkorasik committed:
Add notification (draft)

- Add notifications

lkorasik committed:
Add notification

- Add notify command
- Add messages for notify command
- Add DateConverter
- Add DateType
- Remove not used line in NotExistingTaskIndexException
- Add updateFSMState in RequestHandler
- Add setTimer in RequestHandler
- Add getTime in RequestHandler
- Add Timer in Task
- Rename EP to ENTRY_POINT
- Add new states to fsm for notifications

ddqof commented on src/main/java/com/core/Core.java:
```diff
@@ -26,7 +27,7 @@ public Core(String file_path) {
         File json_file = new File(USERS_FILE);
         GsonBuilder builder = new GsonBuilder();
-        Gson gson = builder.create();
+        Gson gson = builder.setPrettyPrinting().create();
```
зачем setPrettyPrinting?

lkorasik replied:
Ща уберу. Я хотел сделать так, чтоб оно нормально форматировало json. Но это не сработало

ddqof commented on src/main/java/com/core/RequestHandler.java:
```diff
@@ -70,6 +79,25 @@ else if (isDel && !input.equals(Constants.CANCEL_COMMAND)){
+            else {
+                res = "err";
+            }
+             */
```
Это для чего?

lkorasik committed:
Fix code style

- Remove unused code from RequestHandler
- Remove prettyFromatting from Gson in Core
